src/client/stories/suggestedRoutes/suggestedRouteCard.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import SuggestedRouteCard from "../../components/SuggestedRoutes/SuggestedRouteCard";
+import {
+  withKnobs,
+  text,
+  boolean,
+  number,
+  array,
+  object
+} from "@storybook/addon-knobs";
+
+export function suggestedRouteCard() {
+  return (
+    <SuggestedRouteCard
+      time={text("Time")}
+      price={text("Price")}
+      eta={text("ETA")}
+      sailDays={text("Sail days")}
+      maxWaveHs={text("Max wave HS")}
+      avgSpeed={text("Average speed")}
+      hireCost={text("Hire cost")}
+      consumption={text("Consumption")}
```

Comment:
please check all prop type for all

Reply:
@zkwsk Please let me know if it in necessary to change them?